3scale/porta
# THREESCALE-11006 Quick Start content- Main Page Part 2

**What this PR does / why we need it**:

Now that Quick starts are available, we want to make them more accessible. In order to achieve that, the design team proposed two different actions:

1. Adding a Quick starts button to the dashboard. 
![Captura de pantalla 2024-03-18 a las 16 34 23](https://github.com/3scale/porta/assets/72191439/e0887898-9483-4f9e-828f-36f8f3f289ca)

2. When users log in for the first time, they should receive a notification informing them that Quick starts are available and encouraging them to take a tour.
![Captura de pantalla 2024-03-18 a las 16 41 19](https://github.com/3scale/porta/assets/72191439/b35c0166-3c9f-4a89-ad07-7c9d59779d64)

**Which issue(s) this PR fixes** 

[THREESCALE-11006](https://issues.redhat.com/browse/THREESCALE-11006)

**Verification steps** 

1. Go to the dashboard and confirm there is a Quick starts button.
2. Click the button and confirm you're redirected to `/p/admin/quickstarts`.
3. Create a new provider.
4. Confirm that on login you receive the Quick starts notification.
5. Confirm that after logout and login again you don't receive the Quick starts notification.
